docs/build.md

```diff
@@ -74,6 +75,7 @@ The `Build` definition supports the following fields:
 - Optional:
   - `spec.parameters` - Refers to a list of `name-value` that could be used to loosely type parameters in the `BuildStrategy`.
   - `spec.dockerfile` - Path to a Dockerfile to be used for building an image. (_Use this path for strategies that require a Dockerfile_)
+  - `spec.sources` - [Sources](#Sources) describes a slice of artifacts that will be imported into project context, before the actual build process starts.
```

Comment:
Separately, should we add a deprecation notice on `source` ?

Reply:
I think we should head to the deprecation of `source` field, lets tackle that chance in a dedicated pull-request coming next based on #652 .